docs/services/devportal.md

```diff
@@ -0,0 +1,154 @@
+# Developer Portal
+
+The [Developer Portal](https://developer.cscs.ch) facilitates CSCS users to manage client applications' subscriptions to an API at CSCS, such as [FirecREST][ref-firecrest] or [CI/CD][ref-cicd].
+
+## Terminology
+
+### API
+
+An **API** is an interface exposed by CSCS to their users to allow them accessing protected resources (computing, storage, accounting data, etc.) in an automated fashion.
+
+Examples of automated access to HPC resources are:
+
+- A cronjob that triggers a CI/CD pipeline to test software periodically
+- A workflow manager that executes workflows asynchronously
+- A web application that performs unattended data transfer after a job finishes
+
+[](){#ref-devportal-application}
+### Application
+
+An **application** (or "*client application*" or "*client*") is a software created by users that consumes the resources exposed by an API.
+
+At CSCS, it requires [OIDC/OAuth2 access tokens](https://datatracker.ietf.org/doc/html/rfc6749#section-1.4) to authenticate against an [Identity Provider](#identity-provider-idp) (IdP) in order to access the protected resources.
+
+An application *consumes* API resources by *subscribing* to the API. At CSCS, applications can be subscribed to multiple APIs.
+
+### Identity Provider (IdP)
+
+An **identity provider** is an entity that authenticates an application and allows accessing protected resources. In this case, CSCS provides an IdP for all its APIs.
+
+### Production Keys
+
+Applications must identify against the CSCS IdP. To do this, users must create the **production keys** (*Consumer ID* - or *Client ID* -, and *Consumer Secret* - or *Client Secret*) that are going to be used to authenticate.
+
+!!! warning
+    The pair Consumer Key and Secret represents a personal credential to access CSCS resources, therefore can't be shared and must be stored securely (i.e.: not exposed in public repositories, forums, documentation, etc)
+
+## Getting started
+
+Start by browsing to [developer.cscs.ch](https://developer.cscs.ch), then sign in by clicking the `SIGN-IN` button on the top right hand corner of the page.
+
+Once logged in, you will see a list of APIs that are available to your user.
+
+!!! warning
+    You might not see version 1 or version 2 of some APIs. You will be able to see all the versions when you *subscribe* your Application to the API.
+
+### Creating an Application
+
+Click on the `Applications` option at the top of the screen to manage your applications.
+
+![FirecREST Main Page](../images/services/devportal-apis.png)
+
+To create a new application, click on the `ADD NEW APPLICATION` button at the top of the Applications view
+
+![Create New App](../images/services/devportal-create-new-app.png)
+
+Complete the mandatory fields (marked with `*`) and make sure to give the application a unique name and select the number of requests per minute.
+When finished, click on the `SAVE` button.
+
+![Complete New App](../images/services/devportal-complete-new-app.png)
```

Comment:
This image probably isn't needed, because the `Create an application` page is simple and the instruction to `Complete the mandatory fields` instruction is clear.